docs/en/getting-started/get-started-docker.asciidoc

```diff
@@ -121,26 +121,13 @@ endif::[]
 TIP: You might need to scroll back a bit in the terminal to view the password
 and enrollment token.
 
-. Copy the generated password and enrollment token and save them in a secure
-location. These values are shown only when you start {es} for the first time.
-
-. Locate the CA certificate in your container. This command returns the path
-to the `http_ca.crt` certificate, which you'll use in the next step.
-+
-[source,sh]
-----
-docker exec -it es01 /bin/bash -c "find /usr/share/elasticsearch -name http_ca.crt"
-----
```

Comment:
@jkakavas, did you mean to remove these steps? I believe that users should still copy the generated password and enrollment token, and the command to locate the CA cert is useful. Let me know if this wasn't your intention, or if something was altered that necessitated this change.

Reply:
Thanks for catching this, it was simply an over-jealous selection. I only meant to remove the ". Locate the CA cert.." step. This was only useful so that we could find the path in the container and then use it below in the `docker cp` command. Now, we _know_ the path in the container so we can reference it directly in the `docker cp` below